compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmTypeGen.java

```diff
@@ -288,8 +288,8 @@ private static void createTypesInstance(ClassWriter cw, List<BIRTypeDefinition>
                     addObjectFields(mv, objectType.fields);
                     BObjectTypeSymbol objectTypeSymbol = (BObjectTypeSymbol) objectType.tsymbol;
                     addObjectInitFunction(mv, objectTypeSymbol.generatedInitializerFunc, objectType, indexMap,
-                            "$__init$", "setGeneratedInitializer", symbolTable);
-                    addObjectInitFunction(mv, objectTypeSymbol.initializerFunc, objectType, indexMap, "__init",
+                            "$init$", "setGeneratedInitializer", symbolTable);
```

Comment:
@Kishanthan, please note this PR will be removing the `__` part from the init functions.

Reply:
ack, as long as it does not break anything :). pl check for other places in the codegen as well.

Reply:
@Kishanthan All other places were updated.